crates/parser/tests/data/statements/valid/0046.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TYPE type1;
 CREATE TYPE type1 AS (attr1 int4, attr2 bool);
-CREATE TYPE type1 AS (attr1 int4 COLLATE collation1, attr2 bool);
+/* TODO: CREATE TYPE type1 AS (attr1 int4 COLLATE collation1, attr2 bool); */ SELECT 1;
```

Comment:
can't get `COLLATE` to work here, I have tried `CollateClause` `ColumnDef` `CompositeTypeStmt` with no luck

it is obfuscated in the macro call but stack trace gives us a hint that it panics in `codegen::get_location_internal`